### PR TITLE
Markdown test reports

### LINKED ISSUE #744: Add a automated test passed/skipped/failed report
Discussion with @arnetheduck 
Similar to https://github.com/status-im/nimbus markdown files

### PR DESCRIPTION
**Pull request by zedeus**
It works pretty much the same way Nimbus's json test reporting does, except the suites are defined manually and the test cases use the `timedTest` template. All suites that are run get included in the report, and reports are created based the module that calls `summarizeLongTests` and the `const_preset` (minimal/mainnet). This was a very simple solution and given the TODOs in the code, refactoring is probably needed, so consider this more a proof of concept. 

Depends on https://github.com/status-im/nim-testutils/pull/15
Fixes #744

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -148,6 +148,6 @@
 	url = https://github.com/arnetheduck/nim-sqlite3-abi.git
 [submodule "vendor/nim-testutils"]
 	path = vendor/nim-testutils
-	url = https://github.com/status-im/nim-testutils.git
+	url = https://github.com/zedeus/nim-testutils.git
```
Comment (zah) [inline .gitmodules]: I've merged the testutils PR, so you can get back to using our sub-module.

**Comment by zedeus** [comment]
I was hesitant to do it this way since it wouldn't work when run in parallel, but the ways I tried to do it without globals wasn't really working out and since it's sequential I just went with this

**Comment by arnetheduck** [comment]
isn't the globals use a side effect of how `unittest` itself places everything in a global scope? I ran into issues with this in some other tests as well - or in other words, we'd have to fix `unittest` first before this?

**Comment by zah** [comment]
@arnetheduck, yes, but we already have unittest2 which doesn't have this problem. The first step would be to evaluate if all our usage patterns with `timedTest`, `asyncTest`, etc are transferrable to it.

**Pull request merged**